WcaOnRails/app/models/competition.rb

```diff
@@ -331,7 +311,11 @@ def latitude_radians
   def events
     # See https://github.com/cubing/worldcubeassociation.org/issues/95 for
```

Comment:
nit: I think it makes more sense for this comment to live next to the line of code `eventSpecs.split...`